nlp/env.yml

```diff
@@ -29,7 +33,7 @@ dependencies:
   - numpy=1.16.2=py37hacdab7b_0
   - numpy-base=1.16.2=py37h6575580_0
   - oauthlib=3.0.1=py_0
-  - openssl=1.1.1b=h01d97ff_2
+  - openssl=1.1.1=h1de35cc_0
```

Comment:
Weirdly, I got an error about this package not being found:
```
└─▶ conda env update -f nlp/env.yml --prune
Collecting package metadata: done
Solving environment: failed

ResolvePackageNotFound:
  - openssl==1.1.1=h1de35cc_0
```
It might be a good idea to do a manual reinstall (after you've activated the emote virtual env, `conda install openssl=1.1.1`) and export the yml file anew (`conda env export | grep -v "^prefix: " > nlp/env.yml`)

Reply:
Also heads up, it looks like you forgot to add flask to the dependencies, so I'd `conda install flask` too.